client/my-sites/stats/summary/index.jsx

```diff
@@ -111,7 +111,7 @@ const StatsSummary = React.createClass( {
 						key="countries-summary"
 						path="countryviews"
 						period={ this.props.period }
-						query={ query }
+						query={ merge( statsQueryOptions, query ) }
```

Comment:
should we add an empty object, to avoid mutating `statsQueryOptions` ?  Something like `merge( {},  statsQueryOptions, query )`